.github/CONTRIBUTING.md

```diff
@@ -4,12 +4,24 @@ These are the guidelines for contributing to this repository.
 
 ## Issues
 
-File an issue if you think you've found a bug. Please describe
+Please follow the each template by the type of your issue.
```

Comment:
"Please follow the each template by the type of you issue." should read: "Please use  the template for your type of issue"

Reply:
👍